src/panoramic/cli/connections.py

```diff
@@ -0,0 +1,281 @@
+from typing import Any, Dict, Optional, Tuple
+
+import click
+import hologram
+import yaml
+
+from panoramic.cli.config.storage import read_config, update_config
+from panoramic.cli.errors import ConnectionNotFound
+from panoramic.cli.paths import Paths
+from panoramic.cli.print import echo_info
+
+CONNECTION_KEYS = [
+    'type',
+    'user',
+    'host',
+    'port',
+    'password',
+    'database',
+    'schema',
+    'warehouse',
+    'account',
+    'project',
+    'key_file',
+]
+
+
+def create_connection_command(
+    name,
+    type,
+    user,
+    host,
+    port,
+    password,
+    password_stdin,
+    database,
+    schema,
+    warehouse,
+    account,
+    project,
+    key_file,
+    no_test,
```

Comment:
Can we add type annotations to these methods?

Reply:
Added.